### ISSUE TITLE
Test for security policy in other places than SECURITY.md

### ISSUE THREAD

**Issue by CsatariGergely** [ossf/scorecard#4192]
**Is your feature request related to a problem? Please describe.**

There are lots of projects which are describing their security policy in other places, than a SECURITY.md file.

**Describe the solution you'd like**

Expected content of a security policy should be checked in README.md also. 

**Describe alternatives you've considered**

None

**Additional context**

None

**Comment by CsatariGergely** [comment]
What people think about this? Should we try to address it with a pr?

**Comment by diogoteles08** [comment]
Hey @CsatariGergely! I think this could be a positive change, as many projects indeed have the content of a security policy directly on their README. But I'm not sure if this would be worth a 10/10, because there is also some value on having a dedicated SECURITY.md, as:
1. It's usually where security engineers go to look for this content
2. it's the file GitHub fetches to populate the "Security" tab

That said, I'd be in favor of a PR, but I'd first define a more detailed plan for the resultant scoring.

WDYT, @spencerschrock @raghavkaul

**Comment by spencerschrock** [comment]
Having a SECURITY.md is a well known convention. If we start parsing the README, my thoughts go to detection mechanisms and false positives. 

Do we know how widespread the issue is? And what we'd be looking for in a README?

**Comment by lelia** [comment]
> Having a SECURITY.md is a well known convention. If we start parsing the README, my thoughts go to detection mechanisms and false positives.

I agree with this. The extent to which a `README` mentions security should ideally be limited to a brief sentence linking to the actual `SECURITY.md` policy (similar to how `CONTRIBUTING.md` is treated). GitHub will also autodiscover this file and make the policy available at `/orgname/reponame/security/policy` for any repository ([example](https://github.com/ossf/scorecard/security/policy)) that follows this convention.

That said, I would definitely be in support of an additional check for the presence of `SECURITY-INSIGHTS.yml` (which was previously discussed in #2305 and #2479, and has more stringent requirements for what a security policy should entail.

**Comment by dummdidumm** [comment]
When you using Github you can setup an org-wide security policy by creating a special `.github` repository (example: https://github.com/sveltejs/.github/tree/main). If you have a `SECURITY.md` in there it is basically used for all projects in the org. The score card does not detect this currently.

**Comment by UlisesGascon** [comment]
Hey @dummdidumm!  

Actually, that feature is supported. For example, in [pillarjs/cookies](https://github.com/pillarjs/cookies) we use an org policy:  [Scorecard Visualizer – pillarjs/cookies](https://ossf.github.io/scorecard-visualizer/#/projects/github.com/pillarjs/cookies)  

The issue you’re seeing in sveltejs is related to the timing of the last scan:  
- Last scan: [commit 2344b40](https://github.com/sveltejs/svelte/commit/2344b4052eb80ff745b849da759b0d28d2dcaff6) (2 months ago)  
- Policy creation: [commit 999268a](https://github.com/sveltejs/.github/commit/999268aa319458112483323e6a5934edae334e2a) (3 weeks ago)  
- Report: [Scorecard Visualizer – sveltejs/svelte](https://ossf.github.io/scorecard-visualizer/#/projects/github.com/sveltejs/svelte)  

So once you get a fresher scan, the score should be correct 👍 

If you’d like, I can create a small PR with a workflow to proactively report scorecard results on every commit to the main branch, as we do in pillarjs/cookies ([ref](https://github.com/pillarjs/cookies/blob/master/.github/workflows/scorecard.yml))
